incubator/ark-server/configuration/01-config-deploy.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: ark.heptio.com/v1
+kind: Config
+metadata:
+  namespace: {{ .Values.namespace.backup.name }}
+  name: default
+persistentVolumeProvider:
+  name: {{ required "PLEASE SET VALUE" .Values.configuration.cloudprovider  }}
+  config:
+    {{- if eq .Values.configuration.cloudprovider "gcp" }}
+    project: {{ required "PLEASE SET VALUE" .Values.configuration.project  }}
```

Comment:
Unfortunaly, you can't use required values when there is no default in `values.yaml`. CI will fail because there is currently no way to provide defaults in the CI job.

Reply:
Yes I understand, I'll point out in the Readme that those values must be changed before roll out.

I used the `required` keyword because there are not default values that can be used for project or bucket, they are specific for each setup.